.gitlab-ci/build.yml

```
@@ -1,16 +1,40 @@
 ---
-pipeline image:
+.build: &build
```

```suggestion
.build:
```
You don't need the yaml anchor with the extends syntax